src/functionalTest/java/steps/ClaimStatusSteps.java

```diff
@@ -0,0 +1,85 @@
+package steps;
+
+import net.thucydides.core.annotations.Step;
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+
+public class ClaimStatusSteps {
+
+    private String xml;
+    private Response response;
+    private int randomNumber;
+
+    @Step("Given the soap request body")
+    public String soapRequestBody() {
+        String xmlContent = "";
+        try {
+            xmlContent = new String(Files.readAllBytes(Paths.get(getClass().getClassLoader().getResource("statusUpdate.xml").toURI())));
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+
+        randomNumber = (int)(Math.random() * 10000);  // Generate a random number
```

Comment:
Use the XMLFunctions.generateRandomNumber() method here rather than duplicating the code in it.